src/CourseDashboard.hpp

```diff
@@ -12,8 +12,9 @@ class CourseDashboard
     void deleteUserByNick(std::string nick);
     void retriveUserByNick(std::string nick);
     void updateUser(User & user);
+    bool logIn();
 
 private:
     std::vector<User> users_;
-
+    std::map<std::string, std::string> map_;
```

Comment:
the name map_ is not meaningful

Reply:
corrected:
    std::map<std::string, std::string> usersLoginMap_;